arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+                <message fade-in="0.5s" stay="3s" fade-out="0.5s">
+                    <title>`9BLUE BED DESTROYED</title>
+                    <subtitle>`6Your team lost the round</subtitle>
+                </message>
+                <message text="`7`lYour team lost the round!"/>
+            </switch-scope>
+        </switch-scope>
+    </action>
+    <trigger filter="red-team-win-final" scope="player">
+        <action scope="player">
+            <set var="red_team_win" value="0"/>
+            <set var="blue_team_lose" value="2"/>
+            <set var="gameover" value="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>
+        </action>
+    </trigger>
+    <trigger filter="blue-team-win-final" scope="player">
+        <action scope="player">
+            <set var="blue_team_win" value="0"/>
+            <set var="red_team_lose" value="2"/>
+            <set var="gameover" value="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>
+        </action>
+    </trigger>
+    <trigger filter="red-final" scope="team">
+        <action scope="team">
+            <switch-scope outer="team" inner="match">
+                <action scope="match">
+                    <set var="red_team_win" value="2"/>
+                </action>
+            </switch-scope>
+        </action>
+    </trigger>
+    <trigger filter="blue-final" scope="team">
+        <action scope="team">
+            <switch-scope outer="team" inner="match">
+                <action scope="match">
+                    <set var="blue_team_win" value="2"/>
+                </action>
+            </switch-scope>
+        </action>
+    </trigger>
+    <trigger filter="red-bed-broke" scope="match">
+        <action scope="match">
+                <set var="round_state" value="10"/> <!-- start complete-state --> 
+                <set var="blue_team_score" value="blue_team_score+1"/>
+                <set var="blue_team_win" value="1"/>
+                <set var="red_team_lose" value="1"/>
+        </action>
+    </trigger>
+    <trigger filter="blue-bed-broke" scope="match">
+        <action scope="match">
+                <set var="round_state" value="10"/> <!-- start complete-state --> 
+                <set var="red_team_score" value="red_team_score+1"/>
+                <set var="red_team_win" value="1"/>
+                <set var="blue_team_lose" value="1"/>
+        </action>
+    </trigger>
+    <trigger filter="red-bed-broke-2" scope="match">
+        <action scope="match">
+        </action>
+    </trigger>
+    <trigger filter="blue-bed-broke-2" scope="match">
+        <action scope="match">
+        </action>
+    </trigger>
```

Comment:
```suggestion
```
these are triggering nothing...

Reply:
There is something weird happening with PGM, where if I don't have the red-bed-broke-2 or blue-bed-broke-2 filters, the red-bed-broke-1 or blue-bed-broke-1 filters don't fire.   However, was able to remove the dummy trigger/actions, those aren't needed.

Reply:
it is likely that you need separate offset filters to detect one or the other being broken, as it can be a thing that the 2nd part of the bed breaking doesn't give off an effect. 
However that should be simple to workarround by using something like:
```xml
<any id="red-bed-broke">
    <offset vector="0,10,-41">
        <material>air</material>
    </offset>
    <offset vector="0,10,-42">
        <material>air</material>
    </offset>
</any>
```

Reply:
Noted, thanks.